src/pkg/packaging/dir.proj

```diff
@@ -164,7 +164,11 @@
 
   <Target Name="GenerateCombinedInstallers" DependsOnTargets="InitPackage;GenerateBundles" />
 
-  <Target Name="GenerateNugetPackages" DependsOnTargets="InitPackage" Condition="'$(UsePrebuiltPortableBinariesForInstallers)' == 'false'">
+  <PropertyGroup>
+    <PackageManagedComponents>false</PackageManagedComponents>
+  </PropertyGroup>
+
+  <Target Name="GenerateNugetPackages" DependsOnTargets="InitPackage" Condition="'$(UsePrebuiltPortableBinariesForInstallers)' == 'false' and '$(PackageManagedComponents)' == 'true'">
```

Comment:
@weshaggard my last change was wrong, but this one seems to have done the trick. DependencyModel/PlatformAbstractions packages are no longer being generated, but the .DLL's are still present in Bin.